snowcrash.gyp

```diff
@@ -129,12 +141,7 @@
       'target_name': 'test-libsnowcrash',
       'type': 'executable',
       'include_dirs': [
-        'src',
-        'test',
```

Comment:
`test` is not required, because nothing is included from this dir (or it is included w/ full path) 
`src`is inherited from `direct_dependent_settings` from `libsnowcrash`